### ISSUE TITLE
Problem with archiving in Xcode 

### ISSUE THREAD

**Issue by ennaed11** [ErlichChen/react-native-healthkit#3]
Hello I encountered another error. Linking the module has a problem. Every time i try to archive the app, it throws an error that it can't find the library. It works on my simulator but I can't archive it. I already added the search paths in the build settings but the error still happens. Please help ;-;

![image](https://user-images.githubusercontent.com/37971836/50179400-e74f4400-0341-11e9-99fa-152a77d47216.png)

Edit: I think the problem lies here, I'm not sure why Xcode can't see libreact-native-healthkit.a
![image](https://user-images.githubusercontent.com/37971836/50184422-06ed6900-0350-11e9-9cf5-ebaa3eac4e87.png)

**Comment by ErlichChen** [comment]
Thank you for reporting the issue. Did you try to build lib individually?

**Comment by ennaed11** [comment]
No. I am quite inexperienced with xcode so I'm afraid I might break stuff. But I found a fix. 

- I added react-native-healthkit to Target Dependencies under Build Phases according to [this](https://stackoverflow.com/questions/14947139/xcodebuild-failure-clangerror-no-such-file-or-directoryl) StackOverflow post I read. 
- Then I got a cleaned the project and deleted all the contents of Derived Data

**Comment by ErlichChen** [comment]
Great,  I'll check it tomorrow and comment it on the readme.
